utils/helpers/getLocaleYear.ts

```diff
@@ -1,19 +1,22 @@
-// Imports
 import { LangCode } from "@/utils/types/common";
 
+const BE_AD_DIFFERENCE = 543;
+
+/**
+ * Converts a year between the Buddhist Era (BE) and the Anno Domini (AD) calendar.
+ *
+ * @param locale The locale to convert the year to.
+ * @param year The year to convert.
+ * @param fromType The type of the year to convert from, either "AD" or "BE".
+ *
+ * @returns The converted year.
+ */
 export default function getLocaleYear(
   locale: LangCode,
   year: number,
-  fromType?: "AD" | "BE",
+  fromType: "AD" | "BE" = "AD",
```

Comment:
It’s called a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), which is where you give a parameter a default value for when the function is called without a value for that parameter.

In this example, `fromType` is given the default `"AD"`, so when we don’t pass a value for `fromType`, like so:

```ts
getLocaleYear("th", 2024)
```

That would be the same as:

```ts
getLocaleYear("th", 2024, "AD")
```

Because `"AD"` is the default value for the `fromType` parameter.